game-app/game-core/src/main/java/games/strategy/engine/framework/startup/ui/PlayerTypes.java

```diff
@@ -92,5 +92,10 @@ public Type(final String label) {
      * method will create the corresponding {@code IRemotePlayer} instance.
      */
     public abstract Player newPlayerWithName(String name);
+
+    @Override
+    public String toString() {
+      return label;
+    }
```

Comment:
Alternatively you could create a subclass of `DefaultListCellRenderer` and override the `getListCellRendererComponent` method to use the label instead of toString()

Reply:
That's more complexity, so keeping as is.